R/helpers_knitr_GeneralData.R

```diff
@@ -163,7 +163,7 @@ h_knit_print_select_columns.GeneralData <- function(x, ...) {
 h_knit_print_select_columns.Data <- function(x, ...) {
   x %>%
     tidy() %>%
-    dplyr::select(.data$ID, .data$Cohort, .data$Dose, .data$Tox)
+    dplyr::select(ID, Cohort, Dose, Tox)
```

Comment:
did the `.data$` thing not work? generally that is the preferred way

Reply:
see e.g. https://cran.r-project.org/web/packages/dplyr/vignettes/in-packages.html

Reply:
```
f <- function(x = mtcars) {
  x %>% dplyr::select(.data$mpg, .data$am, .data$wt, .data$cyl)
}
f()
```
gives
```
Warning messages:
1: Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.
ℹ Please use `"mpg"` instead of `.data$mpg`
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was generated. 
2: Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.
ℹ Please use `"am"` instead of `.data$am`
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was generated. 
3: Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.
ℹ Please use `"wt"` instead of `.data$wt`
This warning is displayed once every 8 hours.
Call `lifecycle::last_lifecycle_warnings()` to see where this warning was generated. 
4: Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.
ℹ Please use `"cyl"` instead of `.data$cyl`
This warning is displayed once every 8 hours.
```
The link your provide appears to be written for `dplyr` v1.1.0.

I missed the double quotes.  I will amend.

Reply:
ah interesting - especially that they did not update their CRAN vignette yet

Reply:
Perhaps we should also update `DESCRIPTION` to include `tidyselect (>= 1.2.0)`?

Reply:
> ah interesting - especially that they did not update their CRAN vignette yet

Indeed!

Reply:
yes, sounds good to add that package with minimum version requirement